torch/csrc/jit/passes/quantization.h

```diff
@@ -34,16 +34,17 @@ TORCH_API void InsertObserverNodes(
     std::shared_ptr<Graph>& graph,
     Node* observer_node);
 
-/** \brief Inserts fake-quant nodes.
+/** \brief Inserts quant-dequant nodes.
```

Comment:
Please remove `ExpandFakeQuantNodes` from this file, since we are no longer going the FakeQuant nodes route.